src/definitions/modules/dropdown.less

```diff
@@ -436,7 +436,6 @@ select.ui.dropdown {
         backface-visibility: hidden;
         -webkit-overflow-scrolling: touch;
         border-top-width: 0 !important;
-        width: auto;
```

Comment:
see https://github.com/fomantic/Fomantic-UI/pull/2594/commits/52df34588293a7ceb3d372c9700ac201f0ba979a, maybe there is some typo in the code

Reply:
Change is ok. below `width: @menuMinWidth;` wins all the time